libs/cdk/package.json

```diff
@@ -42,7 +42,7 @@
     "url": "https://github.com/rx-angular/rx-angular.git"
   },
   "peerDependencies": {
-    "@angular/core": "^16.0.0",
+    "@angular/core": "^17.0.0",
```

Comment:
These version ranges seem to cause issues when building against newer angular major versions.

Thoughts on making them a little less restrictive? E.g. `>=17.0.0`

```
npm WARN Could not resolve dependency:
npm WARN peer @angular/core@"^16.0.0" from @rx-angular/cdk@16.0.0
npm WARN node_modules/@rx-angular/cdk
npm WARN   @rx-angular/cdk@"^16.0.0" from the root project
npm WARN   1 more (@rx-angular/template)
```

Reply:
cc @LayZeeDK

Reply:
It's the goal, we chose to sync RxA major version with Angular, this way we can guarantee compatibility, the problem with `>=17.0.0` is that we cannot guarantee compatibility for the next majors, it's a bet.

Reply:
Makes sense. Thanks!